### PR TITLE
first task https://epa.ms/nodejs18-hw1

### PR DESCRIPTION
**Pull request by kartelpol**
configure webpack and babel
set up webpack + nodemon bundling (via onBuildEnd hook)
add taskConfig
create 2 models (User, Product) and config for convinient import all of them

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import * as config from './config/taskInfo.json';
+import { Product, User } from 'models';
+
+console.log(config.name);
```
Comment (deser) [inline app.js]: it's better to use an logging abstraction tool. Like this https://www.npmjs.com/package/winston

Comment (deser) [reply app.js]: why do you need config here, are you going to leverage it?

File: app.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+const product = new Product();
```
Comment (deser) [inline app.js:8]: what are you going to do with these objects?

**Comment by deser** [comment]
Any tests?

File: app.js (hunk 2 above)
Comment (kartelpol) [reply app.js:8]: Probably home task creators invent what to do with this later. As for now it's just log a message from constructor

File: app.js (hunk 1 above)
Comment (kartelpol) [reply app.js]: No idea. It's a part of the task

**Pull request merged**